docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+  --intermediate-file "${ZITI_SIGN_CA_NAME}" \
+  --max-path-len "1"
+```
+
+### Create Server and Client Certs for the Control Plane
+
+Set up some initial values for the server and client certificates.
+
+- `ZITI_NETWORK_COMPONENTS_PKI_NAME` is the key file name.
+- `ZITI_NETWORK_COMPONENTS_ADDRESSES` is a comma-separated list of DNS names to add to the SANs.
+- `ZITI_NETWORK_COMPONENTS_IPS` is a comma-separated list of IPs to add to the SANs.
+
+```
+ZITI_NETWORK_COMPONENTS_PKI_NAME="ziti.network.components"
+ZITI_NETWORK_COMPONENTS_ADDRESSES="localhost,${ZITI_HOSTNAME},some.other.name,and.another.name"
+ZITI_NETWORK_COMPONENTS_IPS="127.0.0.1,127.0.21.71,192.168.100.100"
+```
+
+Now we’ll create the key, client, and server certs for the control plane.
+
+```
+"${ZITI_BIN_DIR}/ziti" pki create key \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}"
+
+"${ZITI_BIN_DIR}/ziti" pki create server \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}" \
+  --server-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-server" \
+  --server-name "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-server" \
+  --dns "${ZITI_NETWORK_COMPONENTS_ADDRESSES}" \
+  --ip "${ZITI_NETWORK_COMPONENTS_IPS}"
+
+"${ZITI_BIN_DIR}/ziti" pki create client \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}" \
+  --client-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-client" \
+  --client-name "${ZITI_NETWORK_COMPONENTS_PKI_NAME}"
+```
+
+### Create Server and Client Certs for the HTTP API
+
+Set up some initial values for the server and client certificates. As with the control plane, these are for the key file name, DNS, and IP SANs. The DNS and IP lists are reused from the control plane cert generation.
+
+```
+ZITI_EDGE_API_PKI_NAME="ziti.edge.controller"
+ZITI_EDGE_API_ADDRESSES="${ZITI_NETWORK_COMPONENTS_ADDRESSES}"
+ZITI_EDGE_API_IPS="${ZITI_NETWORK_COMPONENTS_IPS}"
+```
+
+Now we’ll create the key, client, and server certs for the HTTP API.
+
+```
+"${ZITI_BIN_DIR}/ziti" pki create key \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}"
+  
+"${ZITI_BIN_DIR}/ziti" pki create server \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}" \
+  --server-file "${ZITI_EDGE_API_PKI_NAME}-server" \
+  --server-name "${ZITI_EDGE_API_PKI_NAME}-server" \
+  --dns "${ZITI_EDGE_API_ADDRESSES}" \
+  --ip "${ZITI_EDGE_API_IPS}"
+
+"${ZITI_BIN_DIR}/ziti" pki create client \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}" \
+  --client-file "${ZITI_EDGE_API_PKI_NAME}-client" \
+  --client-name "${ZITI_EDGE_API_PKI_NAME}"
+```
+
+### Update the CA Bundle
+
+Add the CAs to the CA bundle and make a copy for the HTTP API CA bundle.
```

Comment:
explain 'why' this is done. it's important to understand the motivation behind doing this